src/frequenz/sdk/timeseries/battery_pool/_result_types.py

```diff
@@ -26,38 +28,24 @@ class PowerMetrics:
     timestamp: datetime = field(compare=False)
     """Timestamp of the metrics."""
 
-    supply_bound: Bound
-    """Supply power bounds.
-
-    Upper bound is always 0 and will be supported later.
-    Lower bound is negative number calculated with with the formula:
-    ```python
-    working_pairs: Set[BatteryData, InverterData] # working batteries from the battery
-        pool and adjacent inverters
-
-    supply_bound.lower = sum(
-        max(
-            battery.power_inclusion_lower_bound, inverter.active_power_inclusion_lower_bound)
-            for each working battery in battery pool
-            )
-        )
-    ```
+    # pylint: disable=line-too-long
+    inclusion_bounds: Bounds
+    """Inclusion power bounds for all batteries in the battery pool instance.
+
+    This is the range within which power requests are allowed by the battery pool.
+
+    When exclusion bounds are present, they will exclude a subset of the inclusion
+    bounds.
+
+    More details [here](https://github.com/frequenz-floss/frequenz-api-common/blob/v0.3.0/proto/frequenz/api/common/metrics.proto#L37-L91).
```

Comment:
This could now point to the generated docs: https://frequenz-floss.github.io/frequenz-api-common/next/python-reference/frequenz/api/common/metrics_pb2/#frequenz.api.common.metrics_pb2.Metric.system_exclusion_bounds

We could even just use python symbols for cross referencing if we add these object indexes:

```diff
diff --git a/mkdocs.yml b/mkdocs.yml
index e8ddb46..13fe0f1 100644
--- a/mkdocs.yml
+++ b/mkdocs.yml
@@ -105,7 +105,9 @@ plugins:
           import:
             # See https://mkdocstrings.github.io/python/usage/#import for details
             - https://docs.python.org/3/objects.inv
+            - https://frequenz-floss.github.io/frequenz-api-common/v0.3/objects.inv
             - https://frequenz-floss.github.io/frequenz-channels-python/v0.14/objects.inv
+            - https://frequenz-floss.github.io/frequenz-api-microgrid/v0.15/objects.inv
             - https://grpc.github.io/grpc/python/objects.inv
             - https://networkx.org/documentation/stable/objects.inv
             - https://numpy.org/doc/stable/objects.inv
```

```suggestion
    See [`frequenz.api.common.metrics_pb2.Metric.system_exclusion_bounds`][] for more details.
```

But for this to work we need to initialize the microgrid API generated docs and we need to tag `v0.3.1` in the common API so we can have a stable link to the docs (for now only the `next` version is provided), so we can update this in a followup PR.

(as a nice side-effect we can remove the `# pylint: disable=line-too-long` too, probably :D)

Reply:
* https://github.com/frequenz-floss/frequenz-sdk-python/pull/571